js/interface.js

```diff
@@ -25,27 +24,22 @@ var $selectAllCheckbox =  $('.file-cell.selectable');
 
 var appList;
 // This should contain either app/org/folder of current folder
-var currentSelection;
+// eslint-disable-next-line no-undef
 var appIcons = new Map();
 var currentOrganizationId;
 var currentFolderId;
 var currentAppId;
 var currentFolders;
 var currentFiles;
 var restoredItems = 0;
-var counterOrganization;
 var currentSearchResult;
 
-var tetherBox;
-
 var folders = [];
-var apps;
-var organizations;
 var navStack = [];
 var beforeSearchNavStack = [];
 
-var sideBarMinWidth = 240;
-var sideBarMaxWidth = 395;
+// var sideBarMinWidth = 240;
```

Comment:
not in use anymore?

Reply:
Not in use. This is related to adjustable sidebar width, which @hcarneiro commented out -- so these variables are commented out too (instead of removed).